src/main/webapp/app/exercises/shared/slide-toggle/slide-toggle.module.ts

```diff
@@ -0,0 +1,11 @@
+import { NgModule } from '@angular/core';
+
+import { ArtemisSharedModule } from 'app/shared/shared.module';
+import { SlideToggleComponent } from 'app/exercises/shared/slide-toggle/slide-toggle.component';
+
+@NgModule({
+    imports: [ArtemisSharedModule],
+    declarations: [SlideToggleComponent],
+    exports: [SlideToggleComponent],
+})
+export class ArtemisSlideToggleModule {}
```

Comment:
We import `ArtemisSlideToggleModule` in each exercise type since the structured grading instructions feature supports multiple exercise types we would need the slide toggle there and for each additional exercise type, we would like to support. Just like the difficulty-picker or mode-picker components.

Reply:
Not really, I declared it in the shared module so different exercise types could use the slide toggle, thanks for pointing this out.